### PR TITLE
feat(Lazada): Added more domain & Imagekeys

### PR DESCRIPTION
**Pull request by ZzzSleepyy**
## Description 

Added more domain to support URL for other countries and Imagekeys.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/L/Lazada/metadata.json [hunk 1]
```diff
@@ -10,11 +10,17 @@
...
+		"lazada.vn",
+		"www.lazada.vn",
+		"lazada.sg",
+		"www.lazada.sg"
 	],
 	"version": "1.0.1",
-	"logo": "https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/logo.png",
-	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/thumbnail.jpg",
+	"logo": "https://i.ibb.co/LpxnMfj/512x512pngoflazada.png",
+	"thumbnail": "https://www.most2414.com/wp-content/uploads/2020/02/02FEB-Insights-Lazada-Service-Marketplace-MOST2414-Cover.jpg",
```
Comment (theusaf) [inline websites/L/Lazada/metadata.json:23]: These assets appear to be the same
```suggestion
	"logo": "https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/logo.png",
	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/thumbnail.jpg",
```

File: websites/L/Lazada/metadata.json [hunk 2]
```diff
@@ -10,11 +10,17 @@
...
-		"www.lazada.com"
+		"www.lazada.com",
+		"lazada.co",
+		"www.lazada.co",
+		"lazada.vn",
+		"www.lazada.vn",
+		"lazada.sg",
+		"www.lazada.sg"
 	],
 	"version": "1.0.1",
```
Comment (theusaf) [inline websites/L/Lazada/metadata.json:21]: Make sure to change the version

File: websites/L/Lazada/presence.ts [hunk 3]
```diff
@@ -5,14 +5,14 @@ const presence = new Presence({
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-			largeImageKey:
-				"https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/logo.png",
+			largeImageKey: "https://i.ibb.co/LpxnMfj/512x512pngoflazada.png",
```
Comment (theusaf) [inline websites/L/Lazada/presence.ts:8]:
```suggestion
			largeImageKey: 
				"https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/logo.png",
```

File: websites/L/Lazada/presence.ts [hunk 4]
```diff
@@ -60,7 +62,8 @@ presence.on("UpdateData", async () => {
 	if (await presence.getSetting<boolean>("incognito")) {
 		presenceData.details = "Incognito";
 		presenceData.largeImageKey =
-			"https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/logo.png";
+			"https://i.ibb.co/LpxnMfj/512x512pngoflazada.png";
```
Comment (theusaf) [inline websites/L/Lazada/presence.ts:65]:
```suggestion
			"https://cdn.rcd.gg/PreMiD/websites/L/Lazada/assets/logo.png";
```

**Review by theusaf — CHANGES_REQUESTED** [review]
The url assets are the same, so they do not need to be changed

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**